tests/integration/test_bayesian_optimization.py

```diff
@@ -96,6 +99,22 @@ def GPR_OPTIMIZER_PARAMS() -> Tuple[str, List[ParameterSet]]:
                 EfficientGlobalOptimization(AugmentedExpectedImprovement().using(OBJECTIVE)),
                 id="AugmentedExpectedImprovement",
             ),
+            pytest.param(
+                20,
+                EfficientGlobalOptimization(
+                    MonteCarloExpectedImprovement(int(1e3)).using(OBJECTIVE),
+                    generate_continuous_optimizer(100),
+                ),
+                id="MonteCarloExpectedImprovement",
+            ),
+            pytest.param(
+                20,
+                EfficientGlobalOptimization(
+                    MonteCarloAugmentedExpectedImprovement(int(1e3)).using(OBJECTIVE),
+                    generate_continuous_optimizer(100),
+                ),
+                id="MonteCarloAugmentedExpectedImprovement",
+            ),
```

Comment:
Perhaps we dont need a full test for the augmented one?

Reply:
I think we should have at least one just in case someone manages to break it, although you're right probably not both. Should we keep the slow test or the fast test?

Reply:
keep the fast one I think